public/css/index.css

```diff
@@ -2063,6 +2063,7 @@ footer .ui.language .menu {
   color: #A7A7A7;
   background: #fafafa;
   width: 1%;
+  vertical-align: top;
```

Comment:
Please don't edit `index.css` directly, instead edit the appropriate `.less` file